logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -895,3 +903,5 @@ internal class MLSConversationDataSource(
 
     private data class CommitOperationResult<T>(val commitBundle: CommitBundle?, val result: T)
 }
+
+val IDENTITIES_EXPIRATION = 1.seconds
```

Comment:
```suggestion
val IDENTITIES_TTL = 1.seconds
```

Just a suggestion to have this named as the common patter for cache, `time to live`.